Check weak linked symbol before use

pvollan opened the pull request:
#### 2c38428816cc65f16488d1c5d2f64cf517f311b8
<pre>
Check weak linked symbol before use
<a href="https://bugs.webkit.org/show_bug.cgi?id=241933">https://bugs.webkit.org/show_bug.cgi?id=241933</a>
&lt;rdar://95797334&gt;

Reviewed by Chris Dumez.

Check weak linked symbol from the Power log framework before use.

* Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm:
(WebKit::WebProcessPool::registerNotificationObservers):
(WebKit::WebProcessPool::unregisterNotificationObservers):

Canonical link: <a href="https://commits.webkit.org/251812@main">https://commits.webkit.org/251812@main</a>
</pre>

cdumez reviewed (approved):
Looks reasonable, assuming it works.

pvollan committed:
Check weak linked symbol before use
https://bugs.webkit.org/show_bug.cgi?id=241933
<rdar://95797334>

Reviewed by Chris Dumez.

Check weak linked symbol from the Power log framework before use.

* Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm:
(WebKit::WebProcessPool::registerNotificationObservers):
(WebKit::WebProcessPool::unregisterNotificationObservers):

Canonical link: https://commits.webkit.org/251812@main
